prometheus_hardware_exporter/core.py

```diff
@@ -89,6 +89,25 @@ def specifications(self) -> List[Specification]:
             A list of specification.
         """
 
+    @property
+    def failed_metrics(self) -> Iterable[Metric]:
+        """Defines the metrics return when collect func fail.
+
+        Yields:
+            metrics: the internal metrics
+        """
+        name = self.__class__.__name__
+        metric = GaugeMetricFamily(
+            name=f"{name.lower()}_collector_failed",
+            documentation=f"{name} Collector fail to fetch metrics",
```

Comment:
typo: Collector *failed* to fetch metrics

Reply:
Updated